apps/frontend/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Martletplace</title>
```

Comment:
Can we change `Martletplace` to `MartletPlace` to keep it consistent everywhere?

Reply:
My assumption is that wouldn't be capitalized...no?
![image](https://github.com/UVicMartletplace/martletplace/assets/73045936/056d5ef2-271c-472e-84ba-87b52e90e6c3)

Reply:
We had a discussion about it and decided `MartletPlace` wasmore readable. Ever since we've been writing it like that in our docs.